scripts/docker/install-go.bash

```diff
@@ -17,7 +17,8 @@ set -e
 
 curl --silent https://dl.google.com/go/go1.13.4.linux-amd64.tar.gz | tar -C /usr/local -xzf -
 
-go get github.com/golang/protobuf/protoc-gen-go
+export GO111MODULE="on"
+go get github.com/golang/protobuf/protoc-gen-go@v1.3.2 # higher protoc-gen failed on gRPC v1.24.0
```

Comment:
Hi @Yancey1989, could you please verify this fixed version by running no-cached Docker build on our CI?

I am unable reproduce this fix in our internal build, so I turned to update go-grpc version as suggested at https://github.com/grpc/grpc-go/issues/3347#issuecomment-580922023.

Reply:
I think we installed the tool

https://github.com/sql-machine-learning/sqlflow/blob/8f4e707169969b54bf26731655893179ab40c14a/scripts/docker/install-go.bash#L20

Reply:
Tthe latest protoc-gen-go version was updated to `v1.3.3` and it does compatible with grpc with `v1.24.0`:https://github.com/sql-machine-learning/sqlflow/blob/8f4e707169969b54bf26731655893179ab40c14a/go.mod#L46

@tonyyang-svail  I think upgrading  `grpc-go` to the latest is fine, will update this PR.